instance/tests/models/test_openedx_theme_mixins.py

```diff
@@ -184,3 +187,54 @@ def test_get_contrasting_font_color(self, test_colors, mock_consul):
             instance.get_contrasting_font_color(bg_color),
             font_color
         )
+
+    # pylint: disable=too-many-locals
+    @patch(
+        'instance.tests.models.factories.openedx_instance.OpenEdXInstance._write_metadata_to_consul',
+        return_value=(1, True)
+    )
+    def test_v1_config_chosen_when_theme_config_provided(self, mock_consul):
+        """
+        Test that when theme_config is provided, the v1 version of the theme configuratino is generated.
```

Comment:
Nit: `configuration`
```suggestion
        Test that when theme_config is provided, the v1 version of the theme configuration is generated.
```